src/lib/components/input/toggle/index.stories.tsx

```diff
@@ -0,0 +1,191 @@
+
+import { useState } from 'react';
+import { Toggle, ToggleProps } from '.';
+
+const story = {
+  title: 'JSX/Inputs/Toggle',
+  component: Toggle,
+  argTypes: {
+    options: {
+      description: 'Object that contains the possible options for rendering in the input.',
```

Comment:
Since we have a story where the toggle has no text, we could add a sentence like "To create an icon-only toggle, pass an empty string" to clarify what to do in this case (though I imagine it won't be common).